content/docs/dvclive/api-reference/live/log_metric.md

```diff
@@ -0,0 +1,106 @@
+# Live.log_metric()
+
+```py
+ def log_metric(name: str, val: float):
+```
```

Comment:
Minor since it's just one file but is it possible to have the Git diff recon that this is the old content/docs/dvclive/api-reference/live/log.md ? Would be easier to review.

> May need to drop this one from the repo and re-add the previous file, `git commit`, and then `git mv` it to this.